src/Component/Site/ComponentController.php

```diff
@@ -2,9 +2,9 @@
 namespace MichaelT\Component\Site;
 
 use Illuminate\Http\Request;
-use MichaelT\Component\Admin\ComponentController;
+use MichaelT\Component\Admin\ComponentController as AdminComponentController;
```

Comment:
I'd rather not apply these changes in order to streamline the process of generating these classes with stubs, same goes to the Admin part